### PR TITLE
Fixes #1907 - formating content

### LINKED ISSUE #1907: refactor template: privacy.html
Refactor following:
https://github.com/webcompat/webcompat.com/blob/master/webcompat/templates/privacy.html

with:
.grid > .grid-row > .grid-cell.x3

Please be aware to add the placeholders for links as they are in the old templates.
Especially for the JS.

Content will not change here.

### PR DESCRIPTION
**Pull request by zoepage**

### CODE REVIEW

**Comment by miketaylr** [comment]
(I assume this needs rebasing before merging as well.... but r+ just for https://github.com/webcompat/webcompat.com/pull/1941/commits/1e8f7472411451002670dbfc226ef939a16afc69)

**Comment by miketaylr** [comment]
nit: the commit message has a typo, it should be `formatting`

File: webcompat/templates/privacy.html [hunk 1]
```diff
@@ -5,25 +5,73 @@
...
+        </p>
+          </div>
+    </article>
+</section>
+<section class="grid">
+    <article class="grid-row">
+      <div class="grid-cell x3">
+        <h2 class="headline-1 highlight">Contact</h2> 
+        <p>
+                You can find all contact information on <a href="./contact" title="our contact page">our contact page</a>.
```
Comment (denschub) [inline webcompat/templates/privacy.html]: Small little nit: Double indentation here.

Comment (zoepage) [reply webcompat/templates/privacy.html]: We'll gonna run a linter through all of that on the rebase branch later on.

**Pull request merged**